Libraries/Components/Touchable/Touchable.js

```diff
@@ -685,7 +731,7 @@ const TouchableMixin = {
    * @throws Error if invalid state transition or unrecognized signal.
    * @sideeffects
    */
-  _receiveSignal: function(signal, e) {
+  _receiveSignal: function(signal: Signal, e: LayoutEvent) {
```

Comment:
This gets forwarded to `_performSideEffectsForTransition`. So, shouldn't `e` be typed as `PressEvent`?